system-variables.md

```diff
@@ -81,6 +81,13 @@ mysql> SELECT * FROM t1;
 - Default value: ON
 - Controls whether statements should automatically commit when not in an explicit transaction. See [Transaction Overview](/transaction-overview.md#autocommit) for more information.
 
+### datadir
+
+- Scope: NONE
+- Default value: /tmp/tidb
+- This variable indicates the location where data is stored.
+- Typically the value when be in the format of `ip_address:port`, indicating the PD server that TiDB connected to on startup.
```

Comment:
```suggestion
- Typically, the value in the format of `ip_address:port` indicates the PD server that TiDB connects to on startup.
```

Reply:
I made a small edit to your change. Combining with Daniel's feedback we can just say "A value in the format.." since above it clarifies it can be a local path.